tests/shell/shell_recorder/selftest.esr

```diff
@@ -32,10 +35,15 @@ STDOUT: Create a new key user:/test/\/evil\\\/tripleEscaped\\/doubleEscaped/key/
 STDOUT: [section]⏎key="multi⏎line"
 < cat `kdb file $Mountpoint`
 
-WARNINGS: 175,175
+WARNINGS: C03100,C03100
```

Comment:
Interesting. This seems to still cause a failure:

```text
CMD: kdb ls user:/tests
RET: 0
STDERR:  Sorry, 2 warnings were issued ;(
 1: Module mini issued the warning C03100:
        Validation Syntactic: Line 1: '[section]' is not a valid key value pair
 2: Module mini issued the warning C03100:
        Validation Syntactic: Line 3: 'line"' is not a valid key value pair
STDOUT: user:/tests/key
=== FAILED Warnings do not match expected pattern C03100,C03100
```

But the warning codes seem to match your change. Not sure what is going on.